packages/compartment-mapper/src/map-parser.js

```diff
@@ -19,14 +16,17 @@ import { parseExtension } from './extension.js';
  *   ModuleTransform,
  *   ModuleTransforms,
  *   ParseFn,
- *   ParseFnAsync,
+ *   AsyncParseFn,
```

Comment:
I’m choosing to treat this as non-breaking since nothing in practice relies on these type names outside of the compartment mapper, although parsers are part of the public API. We tend to treat them as opaque handles and very much doubt there are parsers implemented outside Compartment Mapper.

Reply:
I can live with it